Firmware/printer.cfg

```diff
@@ -252,16 +238,15 @@ gcode:
 [gcode_macro LOAD_FILAMENT]
 gcode:
    M83                            ; set extruder to relative
-   G1 E280 F1800                  ; quickly load filament to down bowden
-   G1 E30 F300                    ; slower extrusion for hotend path
+   G1 E30 F300                    ; load
    G1 E15 F150                    ; prime nozzle with filament
    M82                            ; set extruder to absolute
     
 [gcode_macro UNLOAD_FILAMENT]
 gcode:
    M83                            ; set extruder to relative
    G1 E10 F300                    ; extrude a little to soften tip
-   G1 E-380 F1800                 ; retract filament completely
+   G1 E-380 F30                   ; retract some, but not too much or it will jam
```

Comment:
Changed the wrong value. 

Keep it as F1800, but change the E value from -380 to -40.

`G1 E-40 F1800                   ; retract filament from the toolhead`